test/services/exposed/error-handler.test.js

```diff
@@ -1,5 +1,17 @@
 const sinon = require('sinon');
 const errorHandler = require('../../../src/services/exposed/error-handler');
+const ForbiddenError = require('../../../src/utils/errors/forbidden-error').default;
+
+class FakePayloadError extends ForbiddenError {
+  constructor(...args) {
+    super(...args);
+
+    this.name = this.constructor.name;
```

Comment:
Do you need this in this tests? I think it's using the constructor's name with your code.

Reply:
🙈